### ISSUE TITLE
0.11.0 shows wrong version number

### ISSUE THREAD

**Issue by petere** [kelseyhightower/confd#365]
The binary at https://github.com/kelseyhightower/confd/releases/download/v0.11.0/confd-linux-amd64 reports the wrong version number:

    $ /usr/local/bin/confd -version
    confd 0.12.0-dev

**Comment by bacongobbler** [comment]
Oops! I've updated them now, so it should show `confd v0.11.0`. I accidentally created the release versions [off master](https://github.com/kelseyhightower/confd/commit/76ac8ee3eb588080abca1ade634c224b3379bdc6) after I bumped the confd version to the next release instead of checking out the v0.11.0 tag before building them. I'll keep this ticket open so that I can reflect this in the documentation for next time. Thanks!
